src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -405,6 +407,17 @@ export default class MutationTable<P extends IMutationTableProps> extends React.
             name: "Cancer Type",
             render:CancerTypeColumnFormatter.makeRenderFunction(this),
             sortBy:(d:Mutation[])=>CancerTypeColumnFormatter.sortBy(d, this.props.studyId, this.props.cancerTypeCache),
+            filter:(d:Mutation[], filterString:string, filterStringUpper:string) => {
+                if (this.props.cancerTypeCache) {
+                    return CancerTypeColumnFormatter.filter(d,
+                        filterStringUpper,
+                        this.props.studyId,
+                        this.props.cancerTypeCache,
+                        this.props.studyToCancerType);
+                } else {
+                    return false;
```

Comment:
at some point maybe we should implement a way to say that no filtering should be done on this column if the cache is missing. maybe return undefined or something